pkgs/by-name/em/emulationstation/sources.nix

```diff
@@ -0,0 +1,21 @@
+{
+  fetchFromGitHub,
+}:
+
+{
+  emulationstation = let
```

Comment:
Why is there a need of this detachment?

Reply:
To not clutter the `package.nix` with 20+ lines of fetchers.

Reply:
Fair enough, though there are way lengthy package definitions.

Might be a good idea to automate the updating.

Reply:
Yes, I know. Arcan uses four fetches afaicr.